test.cc

```diff
@@ -15,5 +15,6 @@ using namespace std;
 int main(int argc, char const *argv[]) {
 
    cout << "Hello Git!!" << endl;
+   cout << "I added a new feature" << endl;
```

Comment:
@abukamail you may want to move this line above line 17. It looks better when outputted.